hedera-node/hedera-addressbook-service-impl/src/test/java/com/hedera/node/app/service/addressbook/impl/test/handlers/AddressBookTestBase.java

```diff
@@ -18,7 +18,9 @@
 
 import static com.hedera.node.app.service.addressbook.impl.AddressBookServiceImpl.NODES_KEY;
 import static com.hedera.node.app.service.mono.pbj.PbjConverter.asBytes;
+import static com.hedera.test.utils.KeyUtils.A_COMPLEX_KEY;
```

Comment:
We should not be adding dependencies on anything in the mono package. You should be able to use `com.hedera.pbj.runtime.io.buffer.Bytes.fromHex` instead of `com.hedera.node.app.service.mono.pbj.PbjConverter.asBytes`

Reply:
fixed